packages/formstr-app/src/components/Header/index.tsx

```diff
@@ -83,8 +71,33 @@ export const NostrHeader = () => {
       onClick: () => {
         window.open("https://geyser.fund/project/formstr", "_blank");
       },
-    },
-  ];
+  };
+
+  const dropdownMenuItems: MenuProps["items"] = [];
+
+  if (pubkey) {
+    dropdownMenuItems.push(
+      {
+        key: "relaySettings",
+        label: "Relay Settings",
```

Comment:
can we just call it settings? since i'm assuming we're going to add more settings on this pane as the code evolves.